parsers/CA_YT.py

```diff
@@ -0,0 +1,119 @@
+import arrow
+from bs4 import BeautifulSoup
+import requests
+
+
+timezone = 'Canada/Pacific'
+
+
+def fetch_production(country_code='CA-YT', session=None):
+    """Requests the last known production mix (in MW) of a given region
+
+    Arguments:
+    country_code       -- ignored here, only information for CA-YT is returned
+    session (optional) -- request session passed in order to re-use an existing session
+    """
+
+    """
+    We are using Yukon Energy's data from
+    http://www.yukonenergy.ca/energy-in-yukon/electricity-101/current-energy-consumption
+
+    Generation in Yukon is done with hydro, diesel oil, and LNG.
+
+    There are two companies, Yukon Energy and ATCO aka Yukon Electric aka YECL.
+    Yukon Energy does most of the generation and feeds into Yukon's grid.
+    ATCO does operations, billing, and generation in some of the off-grid communities.
+
+    See schema of the grid at http://www.atcoelectricyukon.com/About-Us/
+
+    The grid covers about 80% of Yukon's population. The remainder has diesel generators.
+
+    Per https://en.wikipedia.org/wiki/Yukon#Municipalities_by_population
+    of total population 35874 (2016 census), 28238 are in municipalities
+    that are connected to the grid - that is 78.7%.
+
+    We assume that the off-grid uses electricity at roughly the same rate as grid,
+    this assumption is probably a bit pessimistic but not too far off.
+    Off-grid generation is with diesel generators.
+
+    Yukon Energy reports only "hydro" vs "thermal" generation.
+    Per http://www.yukonenergy.ca/ask-janet/lng-and-boil-off-gas,
+    in 2016 the thermal generation was about 50% diesel and 50% LNG.
```

Comment:
My concern here is that using "unknown" (700) when it's known to be some mix of oil and gas (650 and 490) over-estimates the intensity a bit. But maybe it'd prompt the source to specify the breakdown more exactly...